### PR TITLE
More and Better Wallpapers

### PR DESCRIPTION
**Pull request by maxgestic**
Added USARRP style wallpapers!

### CODE REVIEW

File: resources/high_phone/html/js/config.js [hunk 1]
```diff
@@ -96,10 +96,16 @@ Config.Backgrounds = [
     {light: "https://i.imgur.com/4UGj1bn.jpg", dark: "https://i.imgur.com/D0hmRFn.jpg"},
     {light: "https://i.imgur.com/6qrvClJ.jpg", dark: "https://i.imgur.com/efkOcHw.jpg"},
     {light: "https://i.imgur.com/Zoi3eqH.jpg", dark: "https://i.imgur.com/alel8lS.jpg"},
-    {light: "https://i.imgur.com/gv4mK4f.jpg", dark: "https://i.imgur.com/jAvHMiY.jpg"}
+    {light: "https://i.imgur.com/gv4mK4f.jpg", dark: "https://i.imgur.com/jAvHMiY.jpg"},
```
Comment (minipunch) [inline resources/high_phone/html/js/config.js:99]: Lmao with the discord link ???

Wonder if these will ever break 🤔

**Comment by minipunch** [comment]
Think u can send some photos of what the new defaults look like in game

**Comment by maxgestic** [comment]
> Think u can send some photos of what the new defaults look like in game

Why?

File: resources/high_phone/html/js/config.js (hunk 1 above)
Comment (maxgestic) [reply resources/high_phone/html/js/config.js:99]: Not sure tbh I think they should be fine

Comment (maxgestic) [reply resources/high_phone/html/js/config.js:99]: Fixed with https://github.com/minipunch/usarealismrp-fxserver/pull/1297/commits/1ce66c207ecf70c6ef3d125a63df7bd2a94103d2

**Comment by maxgestic** [comment]
image
image
image
image
image
image
image
image
image
image
image

**Review by minipunch — APPROVED** [review]
Nice

**Pull request merged**